### PR TITLE
python3Packages.osmnx: 1.1.2 → 1.2.0

### PR DESCRIPTION
**Pull request by sikmir**
###### Description of changes
[Changelog](https://github.com/gboeing/osmnx/blob/main/CHANGELOG.md)

Requires:
* networkx>=2.8
* Rtree>=1.0

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by fabaff — COMMENTED** [review]
Can we move this into #175805? Over 2.5k rebuilds should not go to `master`.

**Comment by sikmir** [comment]
> Can we move this into #175805? Over 2.5k rebuilds should not go to `master`.

Sure, I guess only #175805 author can move it.

**Comment by SuperSandro2000** [comment]
No, you can do it yourself but I can cherry-pick it this time.

File: pkgs/development/python-modules/networkx/default.nix [hunk 1]
```diff
@@ -2,26 +2,25 @@
...
 , setuptools
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "networkx";
   # upgrade may break sage, please test the sage build or ping @timokau on upgrade
-  version = "2.7.1";
+  version = "2.8.2";
+  disabled = pythonOlder "3.8";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/networkx/default.nix:15]: @timokau @omasanori @raskin @collares

**Comment by sikmir** [comment]
> No, you can do it yourself but I can cherry-pick it this time.

How? I can't commit to python-updates branch.

**Comment by SuperSandro2000** [comment]
You can edit the PR and change the target branch.

**Comment by sikmir** [comment]
> You can edit the PR and change the target branch.

I see. Changed target branch to `python-updates`.

**Comment by collares** [comment]
Thanks for the heads-up! I don't think upgrading networkx from 2.7 to 2.8 will break Sage. Either way, if ofBorg tells me tests are broken I can fix it later.

**Comment by collares** [comment]
@ofborg build sage

**Pull request closed without merging**